draft-song-dns-wireformat-http-01.txt

```diff
@@ -101,18 +102,20 @@ Table of Contents
    [I-D.mohan-dns-query-xml] encoding for DNS data, in this approach
    wire-format data is wrapped with a HTTP header and transmitted on
    port 80 or 443.  This protocol is not designed with web developers or
-   advanced usage in application development in mind.  It simply serves
-   as a sort of DNS VPN, and does not introduce another format of DNS
-   data.
+   advanced usage in application development in mind (although it is
```

Comment:
I don't know. Really as a web developer I wouldn't use our approach - a RESTful API is a much more natural fit. Creating a DNS packet via Javascript and parsing the reply is a lot of work for such a task. The approach that I have taken is to say "yes, it is POSSIBLE", without going any further.